cl/users/management/commands/cl_delete_old_messages.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime, timedelta
+
+from django.utils.timezone import make_aware, utc
+
+from cl.lib.command_utils import VerboseCommand
+from cl.users.models import EmailSent
+
+
+class Command(VerboseCommand):
+    """Command to delete stored messages older than the specified number of
```

Comment:
Small thing, but this would be better if it called things "Emails" throughout (and in the name of the file), instead of "Messages", since "Messages" can refer to other things.

Reply:
Changed, I renamed Messages by Emails on file and strings.